Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php

```diff
@@ -208,16 +228,16 @@ function xyz(
             array(
                 "<?php
     \$this->foo(
-        <<<EOTXTa
+<<<EOTXTa
     heredoc
 EOTXTa
         ,
-        <<<'EOTXTb'
+<<<'EOTXTb'
     nowdoc
 EOTXTb
         ,
-        'foo'
-    );
+'foo'
+);
```

Comment:
>why did you change the indentation here?
I forgot to revert this, thanks :+1: 

>to make the tests pass I assume, lol
>this change does need reverting though, and the implementation fixing

no, the test runs just fine with change reverted,
af3ff6258f22d627991a7f04f2ecaef1e9819995